grep.md

```diff
@@ -0,0 +1,64 @@
+---
+title: Grep
+category: CLI
+layout: 2017/sheet
+updated: 2019-10-04
+---
+
+### Usage
+{: .-prime}
+
+```bash
```

Comment:
nitpick: better use sh than bash since there is bash-specific syntax, it also works in plain sh or zsh. Same remark for all other code blocks.

Reply:
I use `bash` for the code blocks because that's what [prism uses for the syntax highlighting](https://github.com/rstacruz/cheatsheets/blob/master/_parcel/app.js#L5).

Reply:
awesome!